hg-fast-export.py

```diff
@@ -475,7 +475,7 @@ def branchtip(repo, heads):
       break
   return tip
 
-def verify_heads(ui,repo,cache,force,branchesmap):
+def verify_heads(ui,repo,cache,force,force_unnamed_heads,branchesmap):
```

Comment:
First i didn't understand what this flag did, then I figured out that it makes fast export ignore unnamed heads. Name it ignore-unnamed-heads instead. Also document what it does when used without the head2branch plugin.

Also, logically as this change is necessary for using head2branch safely, it should come before the patch introducing head2branch.